Assignment-7/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> PHP - Basic Assignment-7 </title>
+    <link rel="stylesheet" href="./css/index.css">
+</head>
+
+<body>
+    <div class="user-input">
+        <form action="./form.php" method="POST" id="user-data" onsubmit="clearFormFields()">
```

Comment:
Event handlers need to be attached via javascript/jquery only.
No handlers should be attached this way.